package com.caredear;


message TokenData{
    required string uid = 1;
    required string appid = 2;
    required string loging = 3;
    required string aes_string = 4;
}

message TokenResult {
    required int32 result_code = 1;
    optional string error_code = 2;
}
